notebooks/SMAP/2.0 Read and Plot SMAP data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Suggest splitting this into three cells: 1) open the file to return a <strong>File Object </strong>(Lines 1 and 3); 2) list <strong>groups</strong> (Lines 6 to 9); list <strong>Datasets</strong> (lines 14 and 15)</p><p> </p><p>I wonder if looping through the keys is necessary.  https://myhdf5.hdfgroup.org/ is a web-based hdf5 viewer and is a much more convenient way to see the groups and datasets.</p><p> </p><p><code>h5dump</code> is another command line tool we could use. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>